index.js

```diff
@@ -302,13 +303,13 @@ function session(options) {
 
       if (shouldDestroy(req)) {
         // destroy session
-        debug('destroying');
+        debug('destroying session store');
         store.destroy(req.sessionID, function ondestroy(err) {
           if (err) {
             defer(next, err);
           }
 
-          debug('destroyed');
+          debug('session store destroyed');
```

Comment:
this destroys the session, not the store

Reply:
ditto as above